client/src/components/TaskList.js

```diff
@@ -1,44 +1,31 @@
 import React, { Component } from 'react';
+const { default: taskService } = require('../services/task.service.js')
```

Comment:
could this be an ES6 import? e.g.
`import taskService from '../services/task.service.js'`

Reply:
I sure can 😁